autoblocks/_impl/tracer.py

```diff
@@ -133,19 +123,26 @@ def _send_event_unsafe(
         trace_id = trace_id or self._trace_id
         timestamp = timestamp or datetime.now(timezone.utc).isoformat()
 
-        req = self._client.post(
-            url=INGESTION_ENDPOINT,
-            json={
-                "message": message,
-                "traceId": trace_id,
-                "timestamp": timestamp,
-                "properties": merged_properties,
-            },
-            timeout=self._timeout_seconds,
-        )
-        req.raise_for_status()
-        resp = req.json()
-        return SendEventResponse(trace_id=resp.get("traceId"))
+        event_dict = {
+            "message": message,
+            "traceId": trace_id,
+            "timestamp": timestamp,
+            "properties": merged_properties,
+        }
+        try:
+            req = await self._client.post(
+                url=INGESTION_ENDPOINT,
+                json=event_dict,
+                headers=self._client_headers,
+                timeout=self._timeout_seconds,
+            )
+            resp = req.json()
+            return SendEventResponse(trace_id=resp.get("traceId"))
+        except Exception as err:
+            log.error(f"Failed to send event to Autoblocks: {err}", exc_info=True)
+            if AutoblocksEnvVar.TRACER_THROW_ON_ERROR.get() == "1":
+                raise err
+            return SendEventResponse(trace_id=None)
```

Comment:
let's just move this back to `send_event` per https://github.com/autoblocksai/python-sdk/pull/115/files#r1499906670

Reply:
Updated